vm.js

```diff
@@ -1434,6 +1434,11 @@ Object.subclass('Squeak.Image',
     instantiateClass: function(aClass, indexableSize, filler) {
         var newObject = new (aClass.classInstProto()); // Squeak.Object
         var hash = this.registerObject(newObject);
+        if ((aClass.sqClass.hash === 2541) && (aClass.hash === 0)) {
```

Comment:
What's class 2541? Why is it necessary to check? Also, we don't need the parens.